src/docs/markdown/caddyfile/directives/log.md

```diff
@@ -217,6 +225,59 @@ Masks IP addresses in the field using a CIDR mask, i.e. the number of bytes from
 }
 ```
 
+##### query
+
+Marks a field to have one or more actions performed, to manipulate the query part of a URL field. Most commonly, the field to filter would be `uri`. The available actions are:
+
+```caddy-d
+<field> query {
+	delete  <key>
+	replace <key> <replacement>
+	hash    <key>
+}
+```
+
+- **delete** removes the given key from the query.
+- **replace** replaces the value of the given query key with **replacement**. Useful to insert a redaction placeholder; you'll see that the query key was in the URL, but the value is hidden.
+- **hash** replaces the value of the given query key with the first 4 bytes of the SHA-256 hash of the value, lowercase hexadecimal. Useful to obscure the value if it's sensitive, while being able to notice whether each request had a different value.
+
+##### cookie
+
+Marks a field to have one or more actions performed, to manipulate a `Cookie` HTTP header's value. Most commonly, the field to filter would be `request>headers>Cookie`. The available actions are:
+
+```caddy-d
+<field> cookie {
+	delete  <name>
+	replace <name> <replacement>
+	hash    <name>
+}
+```
+
+- **delete** removes the given cookie by name from the header.
+- **replace** replaces the value of the given cookie with **replacement**. Useful to insert a redaction placeholder; you'll see that the cookie was in the header, but the value is hidden.
+- **hash** replaces the value of the given cookie with the first 4 bytes of the SHA-256 hash of the value, lowercase hexadecimal. Useful to obscure the value if it's sensitive, while being able to notice whether each request had a different value.
+
+If many actions are defined for the same cookie name, only the first action will be applied.
+
+##### regexp
+
+Marks a field to have a regular expression replacement applied at encoding time.
+
+```caddy-d
+<field> regexp <pattern> <replacement>
+```
+
+The regular expression language used is RE2, included in Go. See the [RE2 syntax reference](https://github.com/google/re2/wiki/Syntax) and the [Go regexp syntax overview](https://pkg.go.dev/regexp/syntax).
```

Comment:
Good idea to link to these

Reply:
This is copied from all the other places we have regexp